torch/_dynamo/symbolic_convert.py

```diff
@@ -2444,3 +2444,65 @@ def YIELD_VALUE(self, inst: Instruction):
         self.generated_items.append(self.pop())
         # TODO(jansel): figure out why this is needed, it isn't in the docs for YIELD_VALUE
         self.push(ConstantVariable(None))
+
+    def GET_YIELD_FROM_ITER(self, inst):
+        tos = self.stack[-1]
+        if not isinstance(tos, ListIteratorVariable):
+            self.pop()
+            assert tos.has_unpack_var_sequence(self)
```

Comment:
assert seems wrong here, should be unimplemented.

Also `.has_unpack_var_sequence()` does not imply `.items` exists, it implies `.unpack_var_sequence` is implemented.

Reply:
It feels like an assert because getting here with this object feels like an issue in our stack, not a coverage gap?

Reply:
`.has_unpack_var_sequence()` does not mean the original python object isn't iterable, it just means dynamo doesn't know how to unpack it.